app/controllers/api/transformation_mappings_controller.rb

```diff
@@ -10,6 +10,18 @@ def create_resource(_type, _id, data = {})
       raise BadRequestError, "Could not create Transformation Mapping - #{err}"
     end
 
+    def validate_vms_resource(type, id, data = {})
+      transformation_mapping = resource_search(id, type, collection_class(type))
+      (transformation_mapping.validate_vms(data["import"]) || {}).tap do |res|
+        %w(valid_vms invalid_vms conflict_vms).each do |key|
+          next unless res.key?(key)
+          res[key].each do |entry|
+            entry["href"] = normalize_href(:vms, entry["id"]) if entry["href"].blank? && entry["id"].present?
```

Comment:
I'm not sure why this postfix conditional is needed - isn't it part of the contract of the return value of `validate_vms` that it *will* have an id, and that it *won't* have an href? If there are cases where this is needed, this should have a test. Perhaps you could handle this in a follow up PR? Thanks @abellotti !